packages/graphiql/src/utility/StorageAPI.js

```diff
@@ -7,7 +7,9 @@
 
 export default class StorageAPI {
   constructor(storage) {
-    this.storage = storage || window.localStorage;
+    if (typeof window !== 'undefined') {
```

Comment:
This discards the `storage` argument passed to the constructor when on the server side; please fix that:

```suggestion
    this.storage = storage || (typeof window !== 'undefined' ? window.localStorage : null);
```